OpenEbooks/OEImages.xcassets/ArrowDown.imageset/Contents.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
just wondering, do we actually need this for OE? because I thought an account on OE is hard-bound to a specific age group? I may be wrong though.

Reply:
I just add it to OE since the NYPLAgeCheck is included in OE build, I can remove them if you think it's unnecessary

And yes, the age gate can be trigger from the Settings tab on iOS 13+, I will leave a detailed testing steps on the ticket when it's ready for QA

Reply:
it there's no way in the OE code to trigger this code and resources, I think we should remove it. You could also avoid compilation of the code via the OPEN_EBOOKS / SIMPLYE preprocessor macros

Reply:
on the other hand, if it complicates things too much, don't worry about it. Perhaps you can just avoid the resources inclusion.